src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.html

```diff
@@ -0,0 +1,300 @@
+                         id="channel_crash"
+                         formControlName="channel_crash">
+                  <label class="custom-control-label"
+                         for="channel_crash"></label>
+                </div>
+              </div>
+            </div>
+
+            <!-- Channel device -->
+            <div class="form-group row">
+              <label class="cd-col-form-label"
+                     for="channel_device">
+                <ng-container i18n>Device</ng-container>
+                <cd-helper i18n-html
+                           html="Includes information about device metrics like anonymized SMART metrics.">
+                </cd-helper>
+              </label>
+              <div class="cd-col-form-input">
+                <div class="custom-control custom-checkbox">
+                  <input type="checkbox"
+                         class="custom-control-input"
+                         id="channel_device"
+                         formControlName="channel_device">
+                  <label class="custom-control-label"
+                         for="channel_device"></label>
+                </div>
+              </div>
+            </div>
+
+            <!-- Channel ident -->
+            <div class="form-group row">
+              <label class="cd-col-form-label"
+                     for="channel_ident">
+                <ng-container i18n>Ident</ng-container>
+                <cd-helper>
+                  <ng-container i18n>Includes user-provided identifying information about the cluster:</ng-container>
+                  <ul>
+                    <li>Cluster description</li>
+                    <li>Contact email address</li>
+                  </ul>
+                </cd-helper>
+              </label>
+              <div class="cd-col-form-input">
+                <div class="custom-control custom-checkbox">
+                  <input type="checkbox"
+                         class="custom-control-input"
+                         id="channel_ident"
+                         formControlName="channel_ident">
+                  <label class="custom-control-label"
+                         for="channel_ident"></label>
+                </div>
+              </div>
+            </div>
+            <legend>
+              <ng-container i18n>Contact Information</ng-container>
+              <cd-helper i18n>Submitting any contact information is completely optional and disabled by default.</cd-helper>
+            </legend>
+            <div class="form-group row">
+              <label class="cd-col-form-label"
+                     for="contact"
+                     i18n>Contact</label>
+              <div class="cd-col-form-input">
+                <input id="contact"
+                       class="form-control"
+                       type="text"
+                       formControlName="contact"
+                       placeholder="John Doe <john.doe@example.com>">
```

Comment:
Let's choose a more neutral form here:
```suggestion
                       placeholder="Example User <user@example.com>">
```